services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -179,6 +179,14 @@ def compose_spec_submitted(self) -> bool:
             "be used to check if the services were started"
         ),
     )
+    project_network_attached: bool = Field(
+        False,
```

Comment:
can you suggest in the name that is a boolean flag?  e.g. ``is_project_network_attached``